ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -38,6 +38,68 @@ const BugDetailsView = (props) => {
     graphFailureStatus,
   } = props;
 
+  // trim off the timestamp and "TEST-UNEXPECTED-XXX | "
+  function lineTrimmer(input) {
+    if (input === undefined) {
+      return '';
+    }
+    if (typeof input === typeof '') {
+      input = input.split('\n');
+    }
+    const lines = input.map((i) => i.split('\n'));
+
+    const trimmedlines = lines.map((l) => {
+      const parts = l.toString().split(' | ');
+      if (parts.length > 2) {
+        parts.shift();
+      }
+      return parts.join(' | ');
+    });
+    return trimmedlines.join('\n');
+  }
+
+  function lineHash(input) {
+    input = lineTrimmer(input);
+    let trimmedlines = [''];
+    try {
+      trimmedlines = input.split('\n');
+    } catch {
+      trimmedlines = [input];
+    }
+
+    let hash = 0;
+    for (let l = 0; l < trimmedlines.length; l++) {
+      for (let i = 0; i < trimmedlines[l].length; i++) {
+        const char = trimmedlines[l].charCodeAt(i);
+        hash = hash ** 5 - hash + char;
```

Comment:
Logging the hash results for [bug 1775733](https://bugzilla.mozilla.org/show_bug.cgi?id=1775733) returns NaN - should be from an overflow. [SubtleCrypte.digest](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest) could be used to calculate the hash.

Reply:
I have been struggling to get digest() to work properly.  I can follow examples in standalone code blocks to get my desire, but integrating this into BugDetailsView, this seems to result in the drop down box for select has 'value' for each option, and instead of it being a 'hash', it is 'object promise'.

Reply:
I guess the question is how hard should we push on this or how much sloppy hash code do we want to accept?

Reply:
updated this in a much different way.  I have learned a lot here, open to ideas, questions, or criticisms.  Tested locally on a few bugs and it works as expected.  I had to install the babel/core to run locally- maybe a fallout of the jest upgrade as part of taskcluster-client-web ?